features/features.go

```diff
@@ -195,14 +179,6 @@ var (
 					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
-	FeatureGateCloudDualStackNodeIPs = newFeatureGate("CloudDualStackNodeIPs").
-						reportProblemsToJiraComponent("machine-config-operator/platform-baremetal").
-						contactPerson("mkowalsk").
-						productScope(kubernetes).
-						enhancementPR("https://github.com/kubernetes/enhancements/issues/3705").
-						enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-						mustRegister()
-
```

Comment:
This was removed in 1.32: https://github.com/kubernetes/kubernetes/blob/7cd7fea30d054efe4f2cc60073a29d2c618258ec/CHANGELOG/CHANGELOG-1.32.md?plain=1#L962

Should we drop this?

Reply:
I already have a PR open to drop this, but yes, whoever of us gets there first